javascript/extractor/src/com/semmle/js/extractor/FileExtractor.java

```diff
@@ -350,7 +356,7 @@ public static FileType forFile(File f, ExtractorConfig config) {
 
     /** Determine the {@link FileType} for a given file based on its extension only. */
     public static FileType forFileExtension(File f) {
-      String lcExt = StringUtil.lc(FileUtil.extension(f));
+      String lcExt = StringUtil.lc(FileUtil.extension(f)); // TODO: Here, it doesn't recognize .html.erb files
```

Comment:
Is it helpful to keep this comment in? Unless you plan on changing something in the future, I'd suggest dropping `TODO: `

Reply:
Oops, I forgot to remove that 😅